schema/swagger.yml

```diff
@@ -179,32 +179,36 @@ definitions:
   NodeStatus:
     type: object
     required:
-      - syncing
+      - broadhash
+      - consensus
       - height
+      - loaded
       - networkHeight
-      - consensus
-      - broadhash
+      - syncing
     properties:
-      syncing:
-        type: boolean
-        example: false
+      broadhash:
+        type: string
+        example: 258974416d58533227c6a3da1b6333f0541b06c65b41e45cf31926847a3db1ea
+        minLength: 64
+        maxLength: 64
+      consensus:
+        type: integer
+        example: 95
+        minimum: 0
+        maximum: 100
       height:
         type: integer
         example: 123
         minimum: 1
+      loaded:
+        type: boolean
+        example: true
       networkHeight:
         type: integer
         example: 123
-      consensus:
-        type: integer
-        example: 95
-        minimum: 0
-        maximum: 100
-      broadhash:
-        type: string
-        example: 258974416d58533227c6a3da1b6333f0541b06c65b41e45cf31926847a3db1ea
-        minLength: 64
-        maxLength: 64
+      syncing:
+        type: boolean
+        example: false
```

Comment:
Please don't change order of the attributes, that makes difficult to read which attribute is added in current PR.

Reply:
There wasn't a order. I am introducing just one attribute and ordering all of them. It shouldn't be that complicated.